sui-execution/latest/sui-move-natives/src/object_runtime/object_store.rs

```diff
@@ -183,6 +208,22 @@ impl<'a> Inner<'a> {
                     ))
                 }
             };
+        // Find all UIDs inside of the value and update the object parent maps
+        let contained_uids =
+            get_all_uids(child_ty_fully_annotated_layout, obj_contents).map_err(|e| {
```

Comment:
Do we need the fully annotated layout for this? (As opposed to the regular layout, because it seemed like we already had that available when we were loading the child object).

Reply:
We need fully annotated since we need to know what fields are UIDs